package.json

```diff
@@ -141,13 +141,14 @@
     "@babel/preset-env": "^7.20.2",
     "@babel/runtime": "^7.20.13",
     "@graphql-codegen/add": "^4.0.0",
-    "@graphql-codegen/cli": "^3.3.0",
-    "@graphql-codegen/client-preset": "^3.0.1",
-    "@graphql-codegen/typescript": "^3.0.0",
-    "@graphql-codegen/typescript-operations": "^3.0.0",
+    "@graphql-codegen/cli": "^5.0.0",
```

Comment:
Was upgrading all the codegen tools required to get it to work with the super graph?

Reply:
i was originally getting al ot of errors when running codegen, so the first task did was update all the packages. but the main package to make federation work is `@graphql-codegen/typescript-resolvers`